### PR TITLE
Customizing Replaceable component Document

### PR DESCRIPTION
**Pull request by fatihskilic**
resolves https://github.com/volosoft/volo/issues/12356

### CODE REVIEW

**Review by braim23 — APPROVED** [review]

**Review by mahmut-gundogdu — APPROVED** [review]

File: en/themes/lepton-x/angular.md [hunk 1]
```diff
@@ -90,3 +90,235 @@ If everything is ok, you can remove the `@volo/abp.ng.theme.lepton` in package.j
...
+
+- **EmptyLayoutComponent:** The **Minimal** layout that **has no layout components** at all.
+
+The **Layout components** and all the replacable components are predefined in `eThemeLeptonXComponents` as enum.
+
+### How to replace a component
+
+```js
+import { ReplaceableComponentsService } from '@abp/ng.core'; // imported ReplaceableComponentsService
+import { eThemeLeptonXComponents } from '@volo/abp.ng.theme.lepton';   // imported eThemeLeptonXComponents enum
```
Comment (muhammedaltug) [inline en/themes/lepton-x/angular.md]: package name is wrong

File: en/themes/lepton-x/angular.md [hunk 2]
```diff
@@ -90,3 +90,235 @@ If everything is ok, you can remove the `@volo/abp.ng.theme.lepton` in package.j
...
+## Navbar Component
+
+Navbar items are used to add **extra functionality to the Sidebar**. The toolbar is a **vertical bar** that **contains** a group of **Navbar routes**.
+
+![Navbar component](images/angular-navbar.png)
+
+```js
+///...
+this.replaceableComponents.add({
+  component: YourNewNavItemsComponent,
```
Comment (muhammedaltug) [inline en/themes/lepton-x/angular.md]: please change component name

File: en/themes/lepton-x/angular.md [hunk 3]
```diff
@@ -90,3 +90,235 @@ If everything is ok, you can remove the `@volo/abp.ng.theme.lepton` in package.j
...
+## Routes Component
+
+The routes component contains routes in the navbar.
+
+![Routes component](images/angular-navbar-routes.png)
+
+```js
+///...
+this.replaceableComponents.add({
+  component: YourNewNavItemsComponent,
```
Comment (muhammedaltug) [inline en/themes/lepton-x/angular.md]: please change component name

File: en/themes/lepton-x/angular.md [hunk 4]
```diff
@@ -90,3 +90,235 @@ If everything is ok, you can remove the `@volo/abp.ng.theme.lepton` in package.j
...
+## General Settings
+
+- The **general settings component** is used to change your **Appearance** , **Container width** and **Language**. At the **bottom right** is the section that contains the settings.
+
+![General settings](images/leptonx-general-settings.png)
+
+```js
+///...
+this.replaceableComponents.add({
+  component: YourNewNavItemsComponent,
```
Comment (muhammedaltug) [inline en/themes/lepton-x/angular.md]: please change component name

File: en/themes/lepton-x/angular.md [hunk 5]
```diff
@@ -90,3 +90,235 @@ If everything is ok, you can remove the `@volo/abp.ng.theme.lepton` in package.j
...
+## Avatar Component
+
+The **Avatar Component** is your user image that displays on the **Toolbar top right**.
+
+![Avatar Component](images/current-user-image.png)
+
+```js
+///...
+this.replaceableComponents.add({
+  component: YourNewMobileNavbarComponent,
```
Comment (muhammedaltug) [inline en/themes/lepton-x/angular.md]: please change component name

**Pull request merged**